libraries/botbuilder-core/src/botState.ts

```diff
@@ -88,7 +81,7 @@ export class BotState implements PropertyManager {
      * @param force (Optional) If `true` the cache will be bypassed and the state will always be read in directly from storage. Defaults to `false`.
      */
     public load(context: TurnContext, force: boolean = false): Promise<any> {
-        const cached: any = context.turnState.get(this.stateKey) as CachedBotState;
+        const cached: CachedBotState = context.turnState.get(this.stateKey);
```

Comment:
technically this is a code change.  are we ok with this?

Reply:
True but it's just a compile time change. The output code is the same

---
In reply to: [219688302](https://github.com/Microsoft/botbuilder-js/pull/477#discussion_r219688302) [](ancestors = 219688302)